recipes/pandoc-plot/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pandoc-plot" %}
+{% set version = "0.2.1.0" %}
+# Getting SHA256 checksums on windows with Get-FileHash
+# returns checksums with capital letters
+# We set it here and make it into lowercase later
+{% set checksum = "383E6BED3BF548BC1E0D179BF214069507C410CA3C5045B32D39C251BA267FD2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LaurentRDC/{{ name }}/releases/download/{{ version }}/windows-x86_64-{{ version }}.zip
+  sha256: {{ checksum|lower }}
+
+build:
+  skip: True  # [not win]
+  number: 0
+
+requirements:
+  run:
+    - pandoc >=2.9
+
+test:
+  commands:
+    - pandoc-plot --version
+    - pandoc-plot --help
+
+about:
+  home: https://github.com/LaurentRDC/pandoc-plot
+  license: GPL-2.0-or-later
+  license_family: GPL
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
You can simply list the license file name here. Conda build will find it.

Reply:
Fixed in 78938e3